scripts/buyer-burner/verify.ts

```
@@ -0,0 +1,35 @@
+import hre, { ethers } from 'hardhat';
```

ESLint

[error] 'hre' is defined but never used. Allowed unused vars must match /_/u.

(@typescript-eslint/no-unused-vars)